beancmd/beancmd.py

```diff
@@ -49,7 +49,11 @@ def main():
         parser.print_help()
         return 2
 
-    return args.func(args)
+    try:
+        return args.func(args)
+    except ValueError as ve:
+        print('Could not run command: {0}'.format(ve))
+        return 2
```

Comment:
```suggestion
        return ve
```

if it errored before would it just return the error like this? does returning 2 change the possible return code from deeper in the code?

Reply:
Ah, this return value is passed directly to `sys.exit` in the `__main__` block, so its the process exit code.